packages/next/package.json

```diff
@@ -114,6 +114,9 @@
       "optional": true
     }
   },
+  "optionalDependencies": {
+    "sharp": "^0.33.2"
```

Comment:
Should this be pinned? I guess it's weird `sharp` still doesn't have a v1 major, but the caret means "any minor or patch within the `0` major range". That's okay?

Reply:
No, thats not what it means.

The `^` behaves differently when 0 is semver major. 

`^0.33.2` will satisfy `0.33.2` and `0.33.3` but it won't satisfy `0.34.0`

https://runkit.com/embed/5fe0w0ylrje5

https://nodesource.com/blog/semver-tilde-and-caret/#:~:text=MAJOR%20ZERO%20AND%20MINOR%20%3E1

Reply:
Oh that's really weird.